test/lib/mocked-balena-api.ts

```diff
@@ -47,6 +47,11 @@ api.balenaBackend = {
 	},
 };
 
+api.patch('/device/v2/:uuid/state', (req, res) => {
+	console.log(`/device/v2/${req.params.uuid}/state called with `, req.body);
+	return res.status(200).end();
```

Comment:
Can drop this now and just return the `Promise.resolve()` since the arg value is checked in a stub test.